app/Policies/OAuth2AccessTokenPolicy.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Policies;
+
+use App\Models\OAuth2AccessToken;
+use App\Models\User;
+use Illuminate\Auth\Access\HandlesAuthorization;
+
+class OAuth2AccessTokenPolicy
+{
+    use HandlesAuthorization;
+
+    /**
+     * Determine whether the user can view the OAuth2 access token.
+     */
+    public function view(User $user, OAuth2AccessToken $token): bool
+    {
+        return true;
```

Comment:
Should this be only admins? It doesn't really hurt anything having that into but it seems weird to share it unnecessarily.

Reply:
The client policy is also like this. We could restrict `viewAny` to admins if desired.

Reply:
Restricted to admins with `viewAny`